website/docs/server/automatic-persisted-queries/automatic-persisted-queries.md

```diff
@@ -0,0 +1,147 @@
+---
+id: automatic-persisted-queries
+title: Automatic Persisted Queries (APQ)
+---
+
+[APQ is technique created by Apollo](https://www.apollographql.com/docs/apollo-server/performance/apq/) to improve
+GraphQL network performance with zero build-time configuration by sending smaller [GraphQL HTTP requests](https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md),
+a smaller request payload reduces bandwidth utilization and speeds up GraphQL Client loading times.
+
+A persisted query is a parsed and validated query document that is cached on a GraphQL Server, along with a unique identifier (SHA-256 hash), that way,
+Clients can send this identifier instead of his corresponding query which will drastically reduce the request size.
+
+To persist a query, a GraphQL Server must first receive it from a Client, then, subsequent requests can just include the identifier
+instead of the query.
```

Comment:
```suggestion
[APQ is a technique created by Apollo](https://www.apollographql.com/docs/apollo-server/performance/apq/) to improve
GraphQL network performance with zero build-time configuration by sending smaller [GraphQL HTTP requests](https://github.com/graphql/graphql-over-http/blob/main/spec/GraphQLOverHTTP.md).
A smaller request payload reduces bandwidth utilization and speeds up GraphQL Client loading times.

A persisted query is a parsed and validated query document that is cached on a GraphQL Server, along with a unique identifier (SHA-256 hash).
This way clients can send this identifier instead of the corresponding query which will drastically reduce the request size.

To persist a query, a GraphQL Server must first receive it from a Client, then, subsequent requests can just include the identifier
instead of the query.
```